IoTCloudSDK/IoTCloudSDK/IoTCloudAPI.swift

```diff
@@ -380,6 +382,23 @@ public class IoTCloudAPI: NSObject, NSCoding {
 
         return newIotapi
     }
+    
+    func saveToUserDefault(){
+        NSUserDefaults.standardUserDefaults().setObject(NSKeyedArchiver.archivedDataWithRootObject(self), forKey: "IoTCloudAPI")
+        NSUserDefaults.standardUserDefaults().synchronize()
+    }
+
+    public override func isEqual(object: AnyObject?) -> Bool {
```

Comment:
It is default (Framework scope) so it can be used by the extensions. I don't think it needs to be documented.